lib/kennel/syncer.rb

```diff
@@ -294,5 +260,24 @@ def filter_actual!(actual)
         end
       end
     end
+
+    def viewer
+      klass = nil
+
+      case ENV["DIFF"]
+      when "basic"
+        klass = Plans::Viewers::Basic
+      when "delta"
+        klass = Plans::Viewers::Delta
+      when nil
+        klass = Plans::Viewers::Basic
+      else
+        raise InvalidViewerError, "Invalid viewer specified."
+      end
+
+      raise ViewerUnavailableError, "Specified viewer is unavailable." unless klass.available?
```

Comment:
Both these error messages are a bit cryptic, they don't tell the user how they can fix the problem. I wonder if we can provide a bit more help to them?